products/models.py

```diff
@@ -0,0 +1,49 @@
+from django.db import models
+
+from cores.models import TimestampZone
+
+class Product(TimestampZone): 
+    name   = models.CharField(max_length=100)
+    price  = models.DecimalField(max_digits=10, decimal_places=2)
+    grapes = models.CharField(max_length=100)
+    winery = models.ForeignKey('Winery', on_delete=models.CASCADE, related_name='products')
+    type   = models.ForeignKey('Type', on_delete=models.CASCADE, related_name='products')
+
```

Comment:
Product에 Paring 테이블과의 ManyToMany 컬럼을 추가해주세요!
이때, through 속성으로 만들어져 있는 PrdouctParing 테이블을 중간테이블로 사용할 수 있도록 만들어주세요!

```suggestion
parings = models.ManyToMany('Paring', through=..
```

Reply:
넵 말씀하신대로 through='Productpairing'으로 설정완료했습니다..!

Reply:
MtoM 필드를 사용하실 때에는 속성을 복수로 만드셔야 합니다.
그래야 나중에 `product.parings.all()`과 같이 가독성 좋은 형태로 코드를 사용하실 수 있습니다!